detection-rules/attachment_cve_2023_21716_rtf_fonts.yml

```diff
@@ -15,6 +15,12 @@ source: |
             .file_extension in~ ("rtf", "doc", "docx")
             or .file_extension in~ $file_extensions_common_archives
             or .file_extension in~ $file_extensions_macros
+            or (
+                .file_extension is null
+                and .file_type == "unknown"
```

Comment:
Not blocking you, but it sounds like we should add more file type matchers based on the contents? That way we can detect pdf(x), ppt(x), doc(x), etc

Reply:
totally, nothing you need to change here. Just making an observation that it sounds like we're short on some file type matchers in the backend